lib/network_config.ml

```diff
@@ -14,28 +14,42 @@
 
 open Network_interface
 
-open Stdext
-open Fun
-open Xstringext
-
 module D = Debug.Make(struct let name = "network_config" end)
 open D
 
+(* Backport of stdext rtrim using Astring functions *)
+let rtrim s =
```

Comment:
I think that we're going to remove it and use `trim` instead ;)

Reply:
There are a few other uses in other files (`network_utils.ml` and `network_server.ml`), I would be happy to get rid of rtrim ans use trim instead. I'd like to double check if you all agree it's not a problem

Reply:
Not a problem. I think that we used `rtrim` because, at the time, that was the easy and sufficient option (not because we didn't want to trim the left side).